pkg/ccl/testccl/sqlccl/sqlccl_test: TestExplainRedactDDL failed

pkg/ccl/testccl/sqlccl/sqlccl_test.TestExplainRedactDDL [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18684131?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18684131?buildTab=artifacts#/) on release-24.1 @ [02fec02903d493f24fa92e16f50223921be71bd0](https://github.com/cockroachdb/cockroach/commits/02fec02903d493f24fa92e16f50223921be71bd0):

Fatal error:

```
panic: test timed out after 4m57s
running tests:
	TestExplainRedactDDL (4m57s)
	TestExplainRedactDDL/ExplainAnalyze (4m52s)
	TestExplainRedactDDL/ExplainAnalyze/Plan (50s)
	TestExplainRedactDDL/ExplainAnalyze/Plan/Verbose (22s)

```
Stack: 

```
goroutine 26331 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestExplainRedactDDL/Explain/Distsql/Json
=== RUN   TestExplainRedactDDL/Explain/Distsql/Verbose
=== RUN   TestExplainRedactDDL/ExplainAnalyze/Gist/NoFlag
=== RUN   TestExplainRedactDDL/ExplainAnalyze/NoMode
=== RUN   TestExplainRedactDDL/Explain/Gist/Memo
=== RUN   TestExplainRedactDDL/Explain/Plan/Types
    explain_test_util.go:96: encountered non-internal error: pq: index "table_1_col1_3_col%p1_6_idx" is a partial index that does not contain all the rows needed to execute this query
=== RUN   TestExplainRedactDDL/ExplainAnalyze/Vec/NoFlag
=== RUN   TestExplainRedactDDL/ExplainAnalyze/Vec/Env
=== RUN   TestExplainRedactDDL/Explain/Distsql
=== RUN   TestExplainRedactDDL/Explain/Gist/NoFlag
=== RUN   TestExplainRedactDDL/Explain/Opt/Shape
=== RUN   TestExplainRedactDDL/ExplainAnalyze/Ddl/Env
=== RUN   TestExplainRedactDDL/ExplainAnalyze/Opt/Verbose
=== RUN   TestExplainRedactDDL/Explain/Ddl/NoFlag
=== RUN   TestExplainRedactDDL/Explain/Gist/Types
=== RUN   TestExplainRedactDDL/Explain/Vec/Types
    explain_test_util.go:96: encountered non-internal error: pq: index "table_1_col1_3_col%p1_6_idx" is a partial index that does not contain all the rows needed to execute this query
=== RUN   TestExplainRedactDDL/ExplainAnalyze/Ddl/Memo
=== RUN   TestExplainRedactDDL/Explain/Plan/Verbose
    explain_test_util.go:96: encountered non-internal error: pq: index "table_1_col1_3_col%p1_6_idx" is a partial index that does not contain all the rows needed to execute this query
=== RUN   TestExplainRedactDDL/ExplainAnalyze/Gist/Memo
=== RUN   TestExplainRedactDDL/ExplainAnalyze/Opt/Types
=== RUN   TestExplainRedactDDL/Explain/Gist/Viz
=== RUN   TestExplainRedactDDL/Explain/NoMode/Json
=== RUN   TestExplainRedactDDL/Explain/NoMode/Verbose
    explain_test_util.go:96: encountered non-internal error: pq: index "table_1_col1_3_col%p1_6_idx" is a partial index that does not contain all the rows needed to execute this query
=== RUN   TestExplainRedactDDL/Explain/Plan/Shape
    explain_test_util.go:96: encountered non-internal error: pq: index "table_1_col1_3_col%p1_6_idx" is a partial index that does not contain all the rows needed to execute this query
=== RUN   TestExplainRedactDDL/ExplainAnalyze/NoMode/Verbose
    explain_test_util.go:96: encountered non-internal error: pq: index "table_1_col1_3_col%p1_6_idx" is a partial index that does not contain all the rows needed to execute this query
=== RUN   TestExplainRedactDDL/ExplainAnalyze/Opt/NoFlag
=== RUN   TestExplainRedactDDL/Explain
=== RUN   TestExplainRedactDDL/Explain/Distsql/Catalog
=== RUN   TestExplainRedactDDL/Explain/Distsql/Memo
=== RUN   TestExplainRedactDDL/Explain/Gist/Env
=== RUN   TestExplainRedactDDL/ExplainAnalyze/NoMode/Memo
    explain_test_util.go:96: encountered non-internal error: pq: query execution canceled due to statement timeout
    explain_test_util.go:96: encountered non-internal error: pq: query execution canceled due to statement timeout
    explain_test_util.go:96: encountered non-internal error: pq: query execution canceled due to statement timeout
    explain_test_util.go:96: encountered non-internal error: pq: query execution canceled due to statement timeout
    explain_test_util.go:96: encountered non-internal error: pq: query execution canceled due to statement timeout
    explain_test_util.go:96: encountered non-internal error: pq: query execution canceled due to statement timeout
    explain_test_util.go:96: encountered non-internal error: pq: index "table_1_col1_3_col%p1_6_idx" is a partial index that does not contain all the rows needed to execute this query
=== RUN   TestExplainRedactDDL/Explain/Vec/Shape
    explain_test_util.go:96: encountered non-internal error: pq: index "table_1_col1_3_col%p1_6_idx" is a partial index that does not contain all the rows needed to execute this query
=== RUN   TestExplainRedactDDL/Explain/NoMode/NoFlag
    explain_test_util.go:96: encountered non-internal error: pq: index "table_1_col1_3_col%p1_6_idx" is a partial index that does not contain all the rows needed to execute this query
=== RUN   TestExplainRedactDDL/Explain/Opt/Types
=== RUN   TestExplainRedactDDL/ExplainAnalyze/Plan/Verbose
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExplainRedactDDL.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-47070

--- a/pkg/ccl/testccl/sqlccl/BUILD.bazel
+++ b/pkg/ccl/testccl/sqlccl/BUILD.bazel
@@ -2,6 +2,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_test")
 
 go_test(
     name = "sqlccl_test",
+    size = "large",
     srcs = [
         "explain_test.go",
         "gc_job_test.go",

